service/src/test/java/bio/terra/policy/service/pao/PaoServiceTest.java

```diff
@@ -62,6 +63,21 @@ void createPaoTest() throws Exception {
     assertThrows(PolicyObjectNotFoundException.class, () -> paoService.getPao(objectId));
   }
 
+  @Test
+  void listPaosTest() {
+    var objectId = UUID.randomUUID();
+    var objectId2 = UUID.randomUUID();
+
+    paoService.createPao(objectId, PaoComponent.WSM, PaoObjectType.WORKSPACE, new PolicyInputs());
+    paoService.createPao(objectId2, PaoComponent.WSM, PaoObjectType.WORKSPACE, new PolicyInputs());
+
+    // add an extra object id that doesn't exist
+    var paos = paoService.listPaos(List.of(objectId, objectId2, UUID.randomUUID()));
+    assertEquals(2, paos.size());
+    assertTrue(paos.stream().anyMatch(p -> p.getObjectId().equals(objectId)));
+    assertTrue(paos.stream().anyMatch(p -> p.getObjectId().equals(objectId2)));
```

Comment:
This is simpler to write using hamcrest, which this project is already using elsewhere.
```suggestion
    assertThat(paos, containsInAnyOrder(List.of(objectId, objectId2)));
```